pkgs/tools/typesetting/tex/texlive/default.nix

```diff
@@ -189,9 +189,16 @@ let
       )
       ( ''
           mkdir "$out"
-          tar -xf "$src" \
-          --strip-components="$stripPrefix" \
-          -C "$out" --anchored --exclude=tlpkg --keep-old-files
+          if [[ "$tlType"  == "tlpkg" ]]; then
+            tar -xf "$src" \
+              --strip-components=1 \
```

Comment:
Why ignore $stripPrefix?

Reply:
The packages containing the `tlpkg/` folder are always the ones with `$stripPrefix == 1`, because they contain `texmf-dist/` and `tlpkg/`. At any rate, here we need to strip `tlpkg/`.